creator/ingest_runs/tasks/validation_run.py

```diff
@@ -1,79 +1,181 @@
 import logging
+from pprint import pformat, pprint
+
+import pandas
 import django_rq
 from django.conf import settings
 from django.core.files.base import ContentFile
 from django_s3_storage.storage import S3Storage
 
+from kf_lib_data_ingest.etl.extract.utils import Extractor
+from kf_lib_data_ingest.validation.data_validator import (
+    Validator as DataValidator,
+)
+from kf_lib_data_ingest.validation.reporting.markdown import (
+    MarkdownReportBuilder,
+)
+
 from creator.decorators import task
 from creator.ingest_runs.models import ValidationRun, ValidationResultset
+from creator.files.models import extract_config_path, Version
+from creator.analyses.file_types import FILE_TYPES
+from creator.analyses.analyzer import extract_data
 
+ROOT_DIR = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
+EXTRACT_CFG_DIR = os.path.join(ROOT_DIR, "extract_configs", "templates")
 S3_STORAGE = "django_s3_storage.storage.S3Storage"
 
 logger = logging.getLogger(__name__)
 
 
-def validate_file_versions(validation_run):
+def clean_and_map(version):
     """
-    TODO - Replace later. Currently counts to 10 then returns
+    Load tabular data from file version into a DataFrame and then use
+    ingest lib's extraction utility to clean the values and map the column
+    names to the standard set of concepts expected by the validator
+    """
+    logger.info(f"Performing extraction to clean and map file {version}")
 
-    Upload validation results and report files to storage backend
+    # Load ingest extract operations for this file
+    if not version.root_file:
+        raise Exception(
+            f"Validation failed. Version {version} missing root_file"
+        )
+    ec = extract_config_path(version)
+    if not ec:
+        logger.warning(
+            f"⚠️  Could not find an extract config for file version {version} "
+            f"with file type: {version.root_file.file_type}. This file will "
+            "not be included in validation."
+        )
+        return None
+    # Extract file content into a DataFrame
+    try:
+        df = pandas.DataFrame(extract_data(version))
+    except Exception as e:
+        logger.error(
+            "Validation failed! Error in parsing {version} content into "
+            "a DataFrame."
+        )
+        raise
+
+    logger.info(
+        f"Applied extract config {os.path.split(ec)[-1]} for file "
+        f"{version.root_file.file_type}, {version.kf_id}"
+    )
+
+    ext = Extractor()
+    ext.logger = logger
+
+    return ext.extract(df, ec)
+
+
+def validate_file_versions(validation_run):
+    """
+    Load ValidationRun.versions into DataFrames, extract data from each
+    version, clean and map the data before passing it to the validator
     """
-    logger.info(f"Begin validation for {validation_run.versions.all()}")
-    results = {}
+    versions = validation_run.versions.all()
+    logger.info(
+        f"Begin validating file versions: {pformat(list(versions))}"
```

Comment:
I guess I'd have to test it out to verify this. But I think when your list longer than some length, vanilla printing it will wrap the items to the next line, where as  if you use pformat, it'll print each item on a new line. I think the latter is more readable in logs.